kpi/tests/api/v2/test_api_assets.py

```diff
@@ -407,11 +407,19 @@ class AssetsSettingsFieldTest(KpiTestCase):
     URL_NAMESPACE = ROUTER_URL_NAMESPACE
 
     def test_query_settings(self):
-        asset_title = 'asset_title'
-        content = {'settings': [{'id_string': 'titled_asset'}],
-                 'survey': [{'label': 'Q1 Label.', 'type': 'decimal'}]}
+        content = {
+            'settings': [
+                {'id_string': 'titled_asset'}
+            ],
+            'survey': [
+                {
+                    'label': 'Q1 Label.',
+                    'type': 'decimal'
+                }
+            ]
+        }
         self.login('someuser', 'someuser')
-        asset = self.create_asset(None, json.dumps(content), format='json')
+        asset = self.create_asset('', json.dumps(content), format='json')
```

Comment:
is this `''` the `name` of the asset?

Reply:
It seems that `self.client.post` (which I think is `Request.post()` behind the scene) does not support (anymore?) sending `None` among data parameters.
```
data = {
    'name': None,
    ...
}
response = self.client.post(url, kwargs)  # Fails
```
When Asset.Name is `None`, it's saved as an empty string in DB. That's why I changed `None` to an empty string in the test.

Reply:
Good to know. I'm still not sure why it doesn't have a name.